stepper-2.0.md

```diff
@@ -59,42 +59,58 @@ Protocol
 ```
 0  START_SYSEX                             (0xF0)
 1  Stepper Command                         (0x62)
-2  stop command                            (0x01)
+2  zero command                            (0x01)
 3  device number                           (0-9)
 4  END_SYSEX                               (0xF7)
 ```
 
-**Stepper step**
+**Stepper step (relative move)**
+
+Position is specified as a 32-bit unsigned long. Speed is specified as a
+16-bit unsigned int.
+
 ```
 0  START_SYSEX                             (0xF0)
 1  Stepper Command                         (0x62)
 2  step command                            (0x02)
 3  device number                           (0-9)
 4  direction                               (0-1) (0x00 = CW, 0x01 = CCW)
-5  num steps byte1 LSB
-6  num steps byte2
-7  num steps byte3 MSB                     (21 bits (2,097,151 steps max))
-8  num speed LSB                           (steps per second)
-9  num speed MSB
-10 END_SYSEX                               (0xF7)
+5  num steps, bits 0-6
+6  num steps, bits 7-13
+7  num steps, bits 14-20
+8  num steps, bits 21-27
+9  num steps, bits 28-32
+10 speed, bits 0-6                         (steps per second * 1000)
+11 speed, bits 7-13
+12 speed, bits 14-16
+12 END_SYSEX                               (0xF7)
 ```
 
-**Stepper to**
+**Stepper to (absolute move)**
+
 Sets a stepper to a desired position based on the number of steps from the zero position.
+Position is specified as a 32-bit signed long.
+
 ```
 0  START_SYSEX                             (0xF0)
 1  Stepper Command                         (0x62)
 2  to command                              (0x03)
 3  device number                           (0-9)
-4  num target byte1 LSB
-5  num target byte2 MSB
-6  speed LSB                               (steps per second)
-7  speed MSB
-8  [optional] accel LSB                    (acceleration in steps/sec^2)
-9  [optional] accel MSB
-10 [optional] decel LSB                    (deceleration in steps/sec^2)
-11 [optional] decel MSB
-12 END_SYSEX                               (0xF7)
+4  num steps, bits 0-6
+5  num steps, bits 7-13
+6  num steps, bits 14-20
+7  num steps, bits 21-27
+8  num steps, bits 28-32
+9  speed, bits 0-6                         (steps per second * 1000)
+10 speed, bits 7-13
+11 speed, bits 14-16
+12 [optional] accel, bits 0-6              (acceleration in steps/sec^2 * 1000)
```

Comment:
We had already decided to remove accel and decel from here and add them back as separate commands. I've done that, but it doesn't appear to be in GH or on my laptop so it's still back on my workstation at home (I haven't synced yet).